losses/l1/L1.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+
+def L1Loss(y_predicted, y_ground_truth, reduction="mean"):
+    """returns l1 loss between two arrays
+
+    :param y_predicted: array of predicted values
+    :type y_predicted: ndarray
+    :param y_ground_truth: array of ground truth values
+    :type y_ground_truth: ndarray
+    :param reduction: reduction mode, defaults to "mean"
+    :type reduction: str, optional
+    :return: l1-loss
+    :rtype: scalar if reduction is sum or mean, else ndarray
+    """
+    # Calculate the absolute difference array
+    absolute_difference = np.abs(y_predicted - y_ground_truth)
+    # L1 distance is the reduced form of the difference array
+    if reduction == "sum":
+        # Reduction can be done by summing up all the values in the difference array (this is known as "L1-Loss")
+        l1_distance = np.sum(absolute_difference)
+        return l1_distance
+    elif reduction == "mean":
+        # Reduction can also be done by taking the mean (this is known as "Mean Absolute Error")
+        mean_absolute_error = np.mean(absolute_difference)
+        return mean_absolute_error
+    else:
```

Comment:
The function will never go into else condition as `reduction` has a default value, maybe there should be another keyword for returning `absolute_difference`

Reply:
`absolute_difference` can now be obtained by passing `reduction="None"`